docs/validation.md

```diff
@@ -0,0 +1,28 @@
+2. The best matching rule from the image policy is looked up and if `verify: false`, the image is admitted.
+3. Trust data of the image is consulted as described in the next section.
+
+## Validation of trust data
+
+Validation of trust data on a high level boils down to two steps:
+
+1. Get all trusted (i.e. signed) image digests related to the tag or digest of the image.
+2. If there is exactly one, admit the image.
+
+Regarding the latter step, rejection upon no matching trusted digests is obvious. However, Connaisseur also needs to reject the image if there is more than one trusted digest since Connaisseur has no context to choose which of the digests the initial tag should be mutated to. Doing so would potentially constitute changing the code you see deployed in your Kubernetes cluster. This means you cannot tag two different images with the same tag (or you need to scrub your notary server of the previous trust data).
+
+Let's now focus on the integral part of Connaisseur that is how to get all trusted digests for an `image:tag` or `image:digest` combination:
+
+Connaisseur looks up trust data of the image in the `root`, `snapshot`, `timestamp` and `targets` files by querying the API of the notary server. Trust data syntax is validated against [their known schemas](https://github.com/sse-secure-systems/connaisseur/tree/master/connaisseur/res) (for an introduction to the trust data formats see the [presentation of our colleague Philipp Belitz](https://berlin-crypto.github.io/assets/Berlin%20Crypto%20Connaisseur.pdf)). Then, the files' signatures are validated against the pinned root key for the `root` file and against the respective keys validated in previous steps for later files. Connaisseur further gathers trust data of potential delegations linked in the `targets` file.
+
+At this point, Connaisseur is left with a set of potentially disjoint or overlapping sets of trust data. Connaisseur filters the trust data for digests that actually relate to the image in validation.
+
+If the image policy rule that governs the image to be validated does contain a `delegations` field, Connaisseur checks whether one of the delegations' sets of trust data does not contain an entry for the image that is being validated, in which case the request is rejected. Otherwise, Connaisseur builds a set over the union of the digests and proceeds with step 2., i.e. accpeting if the size of the resulting set equals 1.
```

Comment:
`accpeting`

Also, I'd rephrase the first sentence to avoid the negation: "field, Connaisseur makes sure that all delegations' sets of trust data do contain an entry for the image" (don't need to repeat "that is being validated"); "if that is not the case, the request is rejected".